### PR TITLE
Bump EPS to 0.2.1.1

### PR DESCRIPTION
**Pull request by maltokyo**
Not sure what else needs to be changed to actually use this in the main installation?

### CODE REVIEW

File: EPS/0.2.1.1/linuxamd64.Dockerfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 FROM python:3.8.1-slim-buster
 
-ENV EPS_VERSION 0.2.0
+ENV EPS_VERSION 0.2.1.1
 ENV EPS_SHA256 e0931428a5a01801528cd8d05a0fdc5ef708b33d491ebb6721edb9237d30c285
```
Comment (NicolasDorier) [inline EPS/0.2.1.1/linuxamd64.Dockerfile:4]: different version but same hash?

**Comment by NicolasDorier** [comment]
Please do not rename and move the files. The old version should still exist in this repo.

**Comment by maltokyo** [comment]
> Please do not rename and move the files. The old version should still exist in this repo.

Apologies, now I am not sure how to undo this.  Can I start the whole PR from scratch?  Do I just close this one?

File: EPS/0.2.1.1/linuxamd64.Dockerfile (hunk 1 above)
Comment (maltokyo) [reply EPS/0.2.1.1/linuxamd64.Dockerfile:4]: Sorry always for stupid Qs.  How to get from here: https://github.com/chris-belcher/electrum-personal-server/releases/tag/eps-v0.2.1.1 the expected hash?

**Comment by maltokyo** [comment]
Hi @NicolasDorier I don't mind to maintain this versioning, but I would require your support (steps to make this happen)

1. How to find the hash to update it
2. Correct steps to make a bump of a version like this, once I know which files to change (any others?) then I can replicate for next time and not bother you and keep this up to date by myself.

**Pull request closed without merging**